chimedb/data_index/util.py

```diff
@@ -41,18 +41,21 @@
 fname_atmel = "atmel_id.dat"
 
 _fmt_acq = re.compile("([0-9]{8})T([0-9]{6})Z_([A-Za-z0-9]*)_([A-Za-z]*)")
-_fmt_corr = re.compile("([0-9]{8})_([0-9]{4}).h5")
-_fmt_hk = re.compile("([A-Za-z]*)_([0-9]{8}).h5")
-_fmt_hkp = re.compile("hkp_prom_([0-9]{8}).h5")
+_fmt_corr = re.compile("([0-9]{8})_([0-9]{4})\.h5")
+_fmt_hk = re.compile("([A-Za-z]*)_([0-9]{8})\.h5")
+_fmt_hkp = re.compile("hkp_prom_([0-9]{8})\.h5")
 _fmt_atmel = re.compile("atmel_id\.dat")
 _fmt_log = re.compile("ch_(master|hk)\.log")
 _fmt_rawadc = re.compile("rawadc\.npy")
 _fmt_rawadc_hist = re.compile("histogram_chan([0-9]{1,2})\.pdf")
 _fmt_rawadc_spec = re.compile("spectrum_chan([0-9]{1,2})\.pdf")
-_fmt_rawadc_h5 = re.compile("[0-9]{6}.h5")
+_fmt_rawadc_h5 = re.compile("[0-9]{6}\.h5")
 _fmt_raw_gains = re.compile("(gains|gains_noisy)\.pkl")
-_fmt_weather = re.compile("(20[12][0-9][01][0-9][0123][0-9]).h5")
+_fmt_weather = re.compile("(20[12][0-9][01][0-9][0123][0-9])\.h5")
 _fmt_calib_data = re.compile(r"\d{8}\.h5")
+_fmt_misc_tar = re.compile(
+    "([0-9]{8})_([A-Za-z][A-Za-z0-9_+-]*)\.tar(?:\.gz|\.bz2|\.xz)"
```

Comment:
Should we consider making the extension `.misc.tar`? My concern is that it's maybe not obvious what's one of the misc files vs a random tarball. I guess they need to have a 8 digit start, and an underscore, but otherwise the pattern is pretty unconstraining.